service_mesh/service_mesh_support/ossm-collecting-ossm-data.adoc

```diff
@@ -16,4 +16,9 @@ and {ProductName}.
 
 include::modules/about-must-gather.adoc[leveloffset=+1]
 
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
```

Comment:
@jwendell Can you please give a thumbs up on this?

Reply:
It LGTM.

Overall, I think this page is a (outdated?) copy of https://docs.openshift.com/container-platform/4.3/support/gathering-cluster-data.html with the additional information about the `--image` parameter. So, looking ahead, I'm not sure if instead a copy we should just point to the canonical must-gather page...

Reply:
@jwendell The paragraph after the `About the must-gather tool`, until the Prerequisites heading on the ossm page is reused content from the link you posted. It is pulling directly from the OCP content, so when they make a change to that about content, it shows up for us too.

The content after About collecting service mesh data is written and maintained by me. So if it's outdated, I can fix it.